frontends/web/src/routes/buy/moonpay.tsx

```diff
@@ -83,32 +86,40 @@ class Moonpay extends Component<Props, State> {
     return t('buy.info.crypto');
   };
 
+
   public render() {
     const { moonpay, t } = this.props;
-    const { height } = this.state;
+    const { height, iframeLoaded } = this.state;
     const account = this.getAccount();
+
     if (!account || moonpay.url === '') {
       return null;
     }
+
     const name = this.getCryptoName();
     return (
       <div className="contentWithGuide">
         <div className="container">
-          <div className={style.header}>
-            <Header title={<h2>{t('buy.info.title', { name })}</h2>} />
-          </div>
-          <div ref={this.ref} className="innerContainer">
-            <div className="noSpace" style={{ height }}>
-              <Spinner text={t('loading')} />
-              <iframe
-                title="Moonpay"
-                width="100%"
-                height={height}
-                frameBorder="0"
-                className={style.iframe}
-                allow="camera; payment"
-                src={`${moonpay.url}&colorCode=%235E94BF`}>
-              </iframe>
+          <div ref={this.ref} className="innerContainer scrollableContainer">
+            <div className={style.header}>
+              <Header title={<h2>{t('buy.info.title', { name })}</h2>} />
+            </div>
+            <div ref={this.ref} className="innerContainer">
+              <div className="noSpace">
+                {!iframeLoaded && <Spinner text={t('loading')} />}
+                <iframe
+                  onLoad={() => {
+                    this.setState({ iframeLoaded: true });
+                  }}
+                  title="Moonpay"
+                  width="100%"
+                  height={iframeLoaded ? height : 0}
+                  frameBorder="0"
+                  className={style.iframe}
+                  allow="camera; payment"
+                  src={`${moonpay.url}&colorCode=%235E94BF`}>
+                </iframe>
+              </div>
```

Comment:
Also to prevent this UI bug from happening:

<img width="400" alt="Screen Shot 2022-11-28 at 07 21 13" src="https://user-images.githubusercontent.com/28676406/204209466-144934a7-a228-41c5-8710-c70534872129.png">